### PR TITLE
Implement raycasts

### LINKED ISSUE #68: Add raycasts

### PR DESCRIPTION
**Pull request by janhohenheim**
Resolves #68

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description = """
 This crate contains the physical engine of
 the simulation, as well as the objects that reside
 within it"""
-version = "0.6.0"
+version = "0.6.1"
```
Comment (tautropfli) [inline Cargo.toml:7]: Shouldn't version bumps be made in a separate PR?

**Review by tautropfli — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (janhohenheim) [reply Cargo.toml:7]: That's how I did it the [https://github.com/myelin-ai/engine/pull/65](last) few [times](https://github.com/myelin-ai/engine/pull/65) in accordance with @jeremystucki

**Review by tautropfli — APPROVED** [review]
